scripts/build/setup/termux_setup_golang.sh

```diff
@@ -1,8 +1,8 @@
 # Utility function for golang-using packages to setup a go toolchain.
 termux_setup_golang() {
 	if [ "$TERMUX_ON_DEVICE_BUILD" = "false" ]; then
-		local TERMUX_GO_VERSION=go1.21.6
-		local TERMUX_GO_SHA256=3f934f40ac360b9c01f616a9aa1796d227d8b0328bf64cb045c7b8c4ee9caea4
+		local TERMUX_GO_VERSION=go1.22.0
+		local TERMUX_GO_SHA256=f6c8a87aa03b92c4b0bf3d558e28ea03006eb29db78917daec5cfb6ec1046265
 		if [ "$TERMUX_PKG_GO_USE_OLDER" = "true" ]; then
 			TERMUX_GO_VERSION=go1.20.13
```

Comment:
Older go should also be bumped to 1.21.x

Reply:
@licy183 Done